src/plugins/login/azureLoginPlugin.test.ts

```diff
@@ -52,6 +52,7 @@ describe("Login Plugin", () => {
   });
 
   it("calls login if azure credentials are not set", async () => {
+    unsetServicePrincipalEnvVariables();
```

Comment:
This test was failing for some reason... Not sure how it got through the checks

Reply:
huh, that's weird, I thought it had been added in the tests, good thing they work now then